lerobot/common/policies/act/modeling_act.py

```diff
@@ -200,25 +200,28 @@ def __init__(self, config: ACTConfig):
         self.config = config
         # BERT style VAE encoder with input tokens [cls, robot_state, *action_sequence].
         # The cls token forms parameters of the latent's distribution (like this [*means, *log_variances]).
+        self.has_state = "observation.state" in config.input_shapes
+        self.latent_dim = config.latent_dim
```

Comment:
Why this addition? I've been trying to avoid binding config attributes to bare instance attributes because now there are two ways to access the same param: `self.latent_dim` and `self.config.latent_dim`. I prefer the latter as it's directly traceable to the config documentation and needs no intermediate step in figuring out where it comes from.

Reply:
Just noticed I had this line below. But yeah, I think we should not have it at all.